src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -85,7 +85,7 @@ public function getPersistedFilterParameters()
         $employee = Context::getContext()->employee;
         $employeeId = $employee->id ?: 0;
 
-        $cachedFilters = $this->cache->getItem("app.product_filters_${employeeId}");
+        $cachedFilters = $this->cache->getItem("app.product_filters_{$employeeId}");
```

Comment:
Oh that's weird I as expecting this to change the output Like with `$employeeId = 42;`
- previous implement would return `app.product_filters_42`
- new implementation return `app.product_filters_{42}`

But after a test they are equivalent, although I'm not a fan of the second one...

Reply:
@jolelievre FYI : https://php.watch/versions/8.2/$%7Bvar%7D-string-interpolation-deprecated